src/device/mod.rs

```diff
@@ -291,6 +294,9 @@ pub struct EventPublishers {
 
     /// Used to manually trigger a nurse event collection
     nurse_collection_trigger_publisher: Option<Tx<Box<()>>>,
+
+    /// Passed down the PQ key rotation task
+    post_quantum: chan::Tx<telio_pq::Event>,
```

Comment:
Nit, you can ignore this if you think it does not make sense:
```suggestion
    post_quantum_publisher: chan::Tx<telio_pq::Event>,
```

Maybe lets name it a publisher, so it more clearly states that this is not the post_quantum entity itself?

Reply:
Done